src/main/java/org/jabref/logic/cleanup/ISSNCleanup.java

```diff
@@ -4,11 +4,11 @@
 import java.util.List;
 import java.util.Optional;
 
-import org.jabref.logic.identifier.ISSN;
 import org.jabref.model.FieldChange;
 import org.jabref.model.cleanup.CleanupJob;
 import org.jabref.model.entry.BibEntry;
 import org.jabref.model.entry.FieldName;
+import org.jabref.model.entry.identifier.ISSN;
```

Comment:
I have put it there since other identifiers (ArXiv and MathSciNet) as well as other "field type classes" like month, authors and keywords are also placed there.